Source/Core/DolphinWX/LogConfigWindow.cpp

```diff
@@ -25,18 +25,11 @@ LogConfigWindow::LogConfigWindow(wxWindow* parent, wxWindowID id)
               _("Log Configuration")),
       enableAll(true)
 {
-  Bind(wxEVT_CLOSE_WINDOW, &LogConfigWindow::OnClose, this);
-  SetMinSize(wxSize(100, 100));
   m_LogManager = LogManager::GetInstance();
   CreateGUIControls();
   LoadSettings();
 }
 
-void LogConfigWindow::OnClose(wxCloseEvent& event)
-{
-  SaveSettings();
```

Comment:
Just brought the old code back in https://github.com/dolphin-emu/dolphin/pull/4292